bot/exts/evergreen/trivia_quiz.py

```diff
@@ -111,23 +318,39 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
                         done_question.append(question_dict["id"])
                         break
 
-                q = question_dict["question"]
-                answer = question_dict["answer"]
+                if "dynamic_id" not in question_dict:
+                    q = question_dict["question"]
```

Comment:
I think it would be better if q was named more aptly like `question`, seeing as that would be more readable and consistent with `answer` being the full word

Reply:
refactored to `question`.